docs/_articles/en/faq.md

```diff
@@ -38,32 +42,32 @@ A: Absolutely not! `sfdx-scanner` can be used on _any_ codebase!
 ## Questions about language support
 #### Q: What languages does `sfdx-scanner` support?
 A: By default, `sfdx-scanner` supports code written in Apex, VisualForce, Java,
-JavaScript, and TypeScript.<br>
+JavaScript, and TypeScript.<br/>
```

Comment:
On my machine, both `<br>` and `<br/>` render correctly. I'm using Chrome. Are you using a different browser?